src/ui/config/index.ts

```diff
@@ -33,6 +35,8 @@ export const Config: IConfig = {
     rawRepoUrl: 'https://raw.githubusercontent.com/ExpressLRS/Backpack',
     srcFolder: '/',
     tagExcludes: [],
+    targetsRepoUrl: 'https://github.com/ExpressLRS/Backpack',
```

Comment:
backpack? is that correct?

Reply:
The backpacks are not extracted out to the targets repo... yet

Reply:
I think this https://github.com/ExpressLRS/Backpac was meant to read https://github.com/ExpressLRS/targets